ui/src/components/dataset/upload/UploadedDatasetDetails.vue

```diff
@@ -1,65 +1,85 @@
 <template>
```

Comment:
Changes:
1. Show metadata selected by the user (project, source raw data, instrument)
2. Provide an input field to allow users to choose dataset name.